cmd/neofs-lens/internal/meta/status.go

```diff
@@ -0,0 +1,48 @@
+package meta
+
+import (
+	"fmt"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var statCMD = &cobra.Command{
+	Use:   "stat",
+	Short: "Object status information",
+	Long:  `Get metabase's indexes related to an object.`,
+	Args:  cobra.NoArgs,
+	Run:   statFunc,
+}
+
+func init() {
+	common.AddAddressFlag(statCMD, &vAddress)
+	common.AddComponentPathFlag(statCMD, &vPath)
+}
+
+func statFunc(cmd *cobra.Command, _ []string) {
+	var addr oid.Address
+
+	err := addr.DecodeString(vAddress)
+	common.ExitOnErr(cmd, common.Errf("invalid address argument: %w", err))
+
+	db := openMeta(cmd, true)
+	defer db.Close()
+
+	res, err := db.ObjectStatus(addr)
+	common.ExitOnErr(cmd, common.Errf("reading object status: %w", err))
+
+	cmd.Printf("Object state: %s\n", res.State)
+	cmd.Printf("Starage's ID: %s\n", res.StorageID)
```

Comment:
can `res.StorageID` be empty? if so, this will be printed unclear

Reply:
yes, it is "peapod" for peapod and empty for FSTree. do you mean the same `<empty>` or smth like that?

Reply:
> `<empty>` or smth like that?

yep, similar to the code below

btw
```suggestion
	cmd.Printf("Storage's ID: %s\n", res.StorageID)
```